src/test/groovy/steps/DataAnalyticsSteps.groovy

```diff
@@ -75,7 +69,7 @@ class DataAnalyticsSteps {
     def http401(count, calls){
         while (count <= calls) {
             def repoName = "generic-dev-local"
```

Comment:
alternative:
```groovy
(count..calls).each{
  repoSteps.deleteRepository(artifactoryURL, repoName, "user1", "password")
  .then().log().ifValidationFails().statusCode(403)
}

Reply:
Done. That's a good suggestion. We should replace all the while's with range each.